src/components/CreateShoppingListForm/CreateShoppingListForm.css

```diff
@@ -4,3 +4,12 @@
 .CreateShoppingListForm {
   color: black;
 }
+
+.selected-store {
```

Comment:
What's the difference between `.selected-store` and `.store-result`?

Reply:
No difference really, they just reside in different components technically so I wanted the css to be consistent